step.yml

```diff
@@ -220,9 +228,21 @@ outputs:
         Public Install Page URLs by the artifact's file path.
         Only set it if the *Enable public page for the App?* option was *enabled*.
 
-        The format is `KEY1=>VALUE|KEY2=>VALUE`
+        The default format is `KEY1=>VALUE|KEY2=>VALUE` but is controlled by the `public_install_page_url_map_format` input
 
         Examples:
 
         - $BITRISE_DEPLOY_DIR/ios_app.ipa=>https://ios_app/public/install/page
         - $BITRISE_DEPLOY_DIR/android_app.apk=>https://android_app/public/install/page|$BITRISE_DEPLOY_DIR/ios_app.ipa=>https://ios_app/public/install/page
+  - BITRISE_PERMANENT_DOWNLOAD_URL_MAP:
+    opts:
+      title: Map of filenames and Permanent Download URLs
+      description: |-
+        Permanent Download URLs by the artifact's file path.
```

Comment:
Maybe: "Permanent Download URL generated by the artifacts' file paths. " would that describe the situ well?

Reply:
These are rather "File Path - URL" pairs for the respective artifact.

Reply:
Then something like this: This input creates a permanent Download URL for artifacts. The URL can be shared in any communication channel and it will not expire.